### PR TITLE
Bug 2067868: Update prometheus client_golang from 1.11.0 => 1.11.1

### PR DESCRIPTION
**Pull request by coreydaley**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -67,4 +67,5 @@ replace (
 	k8s.io/mount-utils => k8s.io/mount-utils v0.23.0
 	k8s.io/pod-security-admission => k8s.io/pod-security-admission v0.23.0
 	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.23.0
+	github.com/prometheus/client_golang => github.com/prometheus/client_golang v1.11.1 // CVE-2022-21698
```
Comment (adambkaplan) [inline go.mod:70]: We have an explicit dependency on client_golang in the `require` section above, please change the dependency there.

**Review by adambkaplan — CHANGES_REQUESTED** [review]
/lgtm cancel

File: go.mod (hunk 1 above)
Comment (coreydaley) [reply go.mod]: updated, ptal

**Review by adambkaplan — APPROVED** [review]
/approve

/lgtm

**Pull request merged**